test/utils/TestSuiteRunner.js

```diff
@@ -193,7 +196,11 @@ module.exports = class TestRunner {
                             session.knex.migrate.latest({
                                 directory: seedPath
                             })
-                                .then((...args) => {
+                                .asCallback((err) => {
+
+                                    if (err) {
+                                        return Promise.reject(err);
```

Comment:
What does the rejection represent here?  I would expect to see some callback called with the error.